internal/backends/selector/lib.rs

```diff
@@ -92,3 +92,8 @@ pub fn use_modules() {
     #[cfg(feature = "i-slint-backend-winit")]
     i_slint_backend_winit::use_modules();
 }
+
+pub fn init_backends() {
+    #[cfg(feature = "i-slint-backend-qt")]
+    i_slint_backend_qt::init();
+}
```

Comment:
But this should take care of the fact that the backend can be disabled at runtime with SLINT_BACKEND or with the viewer arguments

Reply:
Yeah I realize that this is nonsense. The right fix for the init problem - that should also fixe the Tests expecting the testing backend - is to merely ensure that a backend/platform is set. No special sauce in Qt needed then. Will do tomorrow.